### PR TITLE
bring persistent storage up to data with values.yaml

### PR DESCRIPTION
**Pull request by jsirianni**
Does this PR include tests? No, this is documentation only.

I had an issue where the readme was giving wrong default values, and incorrect keys. After reviewing values.yaml, I was able to fix my custom values. This PR resolves that issue for persistent storage only. I did not check the entire readme. 

resolves https://github.com/opendistro-for-elasticsearch/opendistro-build/issues/163

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ To install the chart with the release name `my-release`:
 
 `❯ helm install --name my-release opendistro-es-1.1.0.tgz`
 
-The command deploys OpenDistro Kibana and Elasticsearch with its associated components (data statefulsets, masters, clients) on the Kubernetes cluster in the default configuration.
+The command deploys OpenDistro Kibana and Elasticsearch with its associated components (data statefuls
+
+    ets, masters, clients) on the Kubernetes cluster in the default configuration.
```
Comment (peterzhuamazon) [inline helm/README.md:33]: Seems like a typo for newlines?

File: helm/README.md [hunk 2]
```diff
@@ -526,8 +533,13 @@ The following table lists the configurable parameters of the opendistro elastics
...
 | `elasticsearch.data.javaOpts`                             | Elasticsearch data configurable java options to pass to startup script                                                                                   | `"-Xms512m -Xmx512m"`                                                   |
-| `elasticsearch.data.storageClassName`                     | Elasticsearch data storageClassName                                                                                                                      | `"gp2-encrypted"`                                                       |
-| `elasticsearch.data.storage`                              | Elasticsearch data storage                                                                                                                               | `"100Gi"`                                                               |
+| `elasticsearch.data.persistence.enabled`                  | Elasticsearch data enable persistent storage                                                                                                             | `true`                                    1                              |
```
Comment (peterzhuamazon) [inline helm/README.md:536]: "1" Another typo?

**Comment by peterzhuamazon** [comment]
Reference from [Here](https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/opendistro-es/values.yaml)

**Comment by jsirianni** [comment]
Hi @peterzhuamazon. Thank you for the quick review. Apologies on the typos and formatting.  I believe they are resolved.

**Pull request merged**